core/modules/parsers/wikiparser/rules/filteredtranscludeblock.js

```diff
@@ -31,6 +31,16 @@ exports.init = function(parser) {
 
 exports.parse = function() {
 	// Move past the match
+	var filterStart = this.parser.pos + 3;
+	var filterEnd = filterStart + this.match[1].length;
+	var toolTipStart = filterEnd + 1;
+	var toolTipEnd = toolTipStart + (this.match[2] ? this.match[2].length : 0);
+	var templateStart = toolTipEnd + 2;
+	var templateEnd = templateStart + (this.match[3] ? this.match[3].length : 0);
+	var styleStart = templateEnd + 2;
+	var styleEnd = styleStart + (this.match[4] ? this.match[4].length : 0);
+	var classesStart = styleEnd + 1;
+	var classesEnd = classesStart + (this.match[5] ? this.match[5].length : 0);
```

Comment:
We always collapse multiple var statements into one:

```
var filterStart = this.parser.pos + 3,
	filterEnd = .....
```

This applies to all changes throughout this PR

Reply:
Does this apply while there is assignment?

Hope this can be done by eslint.

Reply:
I think this would need to be resolved manually.